src/MongoDB.Driver.Core/Core/Operations/AggregateOperation.cs

```diff
@@ -253,6 +253,7 @@ public bool RetryRequested
         /// <value>
         /// A value indicating whether the server should use a cursor to return the results.
         /// </value>
+        [Obsolete("Newer versions of the server always use a cursor.")]
```

Comment:
Consider using release version for line: https://github.com/rstam/mongo-csharp-driver/pull/142/files#diff-74997c1c57d0c85a3cc0e2914855bc40R349 as per Jeff's comment (https://github.com/MikalaiMazurenka/mongo-csharp-driver/pull/13#discussion_r366664338)

Reply:
Also, please specify version when support ended as per Jeff's comment (https://github.com/MikalaiMazurenka/mongo-csharp-driver/pull/13#discussion_r366664126)

Reply:
Historically we have never gone back and replaced pre-release versions with release versions after the fact. But I've done so in this instance.

I reworded the Obsolete comment to include the server version in which support for UseCursor ended.

Done.